tests/tools/commands/test_venv.py

```diff
@@ -801,6 +801,7 @@ def create_venv_pex():
     assert not os.path.exists(pex_root)
 
 
+@pytest.mark.skipif(sys.version_info < (3,5), reason="py2 doesn't support system-site-packages")
```

Comment:
Please do not skip the test for Python 2.7. Please instead remove the portion of the test that checks for pyvenv.cfg. Python 2.7 _does_ support supprt system site packages.

I feel like I'm not making the key point clear. That is, you don't actually care how system-site-packages get wired. The pyvenv.cfg file is a detail. ALl you care about is that system site packages get _used_ in actuality. The rest of your test test that and that's good.

Reply:
OK. I hope you are right about py2.7 support system site packages.

Reply:
Could you please rerun CI? There are some network errors I suppose.

Reply:
Done. You may want to use `./dtox.sh -epy27 -- test_venv` to cycle on this more quickly on your own machine. That will set up (one time) a docker container with Python 2.7 in it and run tox from inside there against Pythhon 2.7.

Reply:
Command failed
```
$ ./dtox.sh -epy27 -- test_venv
[+] Building 2.1s (4/4) FINISHED                                                                          
 => [internal] load build definition from Dockerfile                                                 0.0s
 => => transferring dockerfile: 522B                                                                 0.0s
 => [internal] load .dockerignore                                                                    0.0s
 => => transferring context: 2B                                                                      0.0s
 => ERROR [internal] load metadata for docker.io/library/a7f7e4652237:latest                         2.0s
 => [auth] library/a7f7e4652237:pull token for registry-1.docker.io                                  0.0s
------
 > [internal] load metadata for docker.io/library/a7f7e4652237:latest:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

Anyway, from the error log I believe py2.7 site doesn't have support for system-site-packages.

Reply:
It absolutely does.

Using your branch, I first make 1 pex and then 2 venvs from it - one using your new `--system-site-packages` option and one not:
```
$ python2.7 -mpex cowsay -ccowsay -o cowsay.pex --include-tools
$ PEX_TOOLS=1 ./cowsay.pex venv --system-site-packages venv.ssp
$ PEX_TOOLS=1 ./cowsay.pex venv venv.no-ssp
```

I then test the sys.path of each:
```
$ PEX_INTERPRETER=1 venv.ssp/pex -c 'import sys; print("\n".join(sys.path))'

/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/lib/python2.7
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/lib/python2.7/plat-x86_64-linux-gnu
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/lib/python2.7/lib-tk
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/lib/python2.7/lib-old
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/lib/python2.7/lib-dynload
/usr/lib/python2.7
/usr/lib/python2.7/plat-x86_64-linux-gnu
/usr/lib/python2.7/lib-tk
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/local/lib/python2.7/site-packages
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp/lib/python2.7/site-packages
/usr/local/lib/python2.7/site-packages
/usr/local/lib/python2.7/dist-packages
/usr/lib/python2.7/dist-packages
$ PEX_INTERPRETER=1 venv.no-ssp/pex -c 'import sys; print("\n".join(sys.path))'

/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/lib/python2.7
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/lib/python2.7/plat-x86_64-linux-gnu
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/lib/python2.7/lib-tk
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/lib/python2.7/lib-old
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/lib/python2.7/lib-dynload
/usr/lib/python2.7
/usr/lib/python2.7/plat-x86_64-linux-gnu
/usr/lib/python2.7/lib-tk
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/local/lib/python2.7/site-packages
/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp/lib/python2.7/site-packages
```

That's a difference of:
```
$ diff <(PEX_INTERPRETER=1 venv.ssp/pex -c 'import sys; print("\n".join(sys.path))' | sed 's|/home/jsirois/dev/pantsbuild/jsirois-pex/venv.ssp||') <(PEX_INTERPRETER=1 venv.no-ssp/pex -c 'import sys; print("\n".join(sys.path))' | sed 's|/home/jsirois/dev/pantsbuild/jsirois-pex/venv.no-ssp||')
13,15d12
< /usr/local/lib/python2.7/site-packages
< /usr/local/lib/python2.7/dist-packages
< /usr/lib/python2.7/dist-packages
```

And that difference is, in fact, the python2.7 (system) site-packages.

Reply:
Seems like pex will inject customized virtual env for python 2.7, and `site.py` is manipulated to detect `no-global-site-packages.txt`.

Reply:
Original test would fail because for the injected venv, its venv_interpreter.site_packages seems broken.

Reply:
@PhoenixHe-TSP let me know if you need any help here, but I'll assume you've used [pyenv](https://github.com/pyenv/pyenv) or similar to get a local test / edit feedback loop going where you use `tox -epy27 -- test_venv`.